src/main.js

```diff
@@ -11,7 +11,7 @@ if (require.main === module) {
 }
 
 export function createInstance(config=cli.argv) {
-  log.level(config.logLevel);
+  log.level = config.logLevel;
```

Comment:
I'm curious if this has changed subtly.

The bunyan method of `log.level()` will return the level if there are not arguments or set it if one is provided. So if `cli.argv.logLevel` is `undefined` then I think it will be a noop, whereas assigning `level` to `undefined` in pino likely causes it to reset to the default.

I'll look into it a bit more and maybe wrap this in an `if`.